src/index.js

```diff
@@ -0,0 +1,114 @@
+import './style.css';
+import '@fortawesome/free-solid-svg-icons';
+
+class Todo {
+  constructor(desc, completed, id) {
+    this.desc = desc;
+    this.completed = completed;
+    this.id = id;
+  }
+}
+class UI {
+  static displayTodos = (todos) => {
+    const ul = document.querySelector('.list');
+    ul.innerHTML = '';
+    todos.forEach((tod) => {
+      ul.innerHTML += `
+        <li id=${tod.id} class="item">
+          <div class='group'>
+            <input id=${tod.id} class='bdan' name='checkbox' type="checkbox" ${
+        tod.completed && 'checked'
+      } />
+            <p id=${tod.id} class='${
+        tod.completed ? 'line' : ''
+      }' contenteditable="true">${tod.desc}</p>
+          </div>
+          <i class="fa-solid fa-trash"></i>
+        </li>`;
+    });
```

Comment:
Great job 
I think it's will be better if you separate your classes and put them inside the modules folder instead of using them in a single file